index.html

```diff
@@ -34,137 +34,215 @@ <h1 class="logo-heading">Fun Bus</h1>
         </div>
-    </div>
-    <div class="row">
-      <div class="col-sm-12">
-        <h2>Destinations</h2>
-        <table class="table table-striped">
-          <thead>
-            <tr>
-              <th scope="col">Location</th>
-              <th scope="col">Activity</th>
-              <th scope="col">Airfare</th>
-              <th scope="col">Hotel</th>
-            </tr>
-          </thead>
-          <tbody>
-            <tr>
-              <th scope="row">Beach</th>
-              <td>Chillin</td>
-              <td>$600</td>
-              <td>$400</td>
-            </tr>
-            <tr>
-              <th scope="row">Mountain</th>
-              <td>Hiking</td>
-              <td>$400</td>
-              <td>$200</td>
-            </tr>
-            <tr>
-              <th scope="row">Island</th>
-              <td>Boating</td>
-              <td>$1700</td>
-              <td>$600</td>
-            </tr>
-          </tbody>
-        </table>
-      </div>
-    </div>
-    <div class="row">
-      <div class="col-sm-4">
-        <h4>Fun In The Sun</h4>
-        <p>Expedition excursion design darn excursion fun, clean simple organized WordPress Travel colorful webdesign. Traveler blogger website design expedition clean excursion traveling.</p>
-        <button type="button" class="btn btn-info">Get More Info</button>
+        <div class="col-sm-6">
+            <img src="img/adventure.jpg" class="img-fluid rounded" alt="Let's go on adventure!">
+        </div>
       </div>
-      <div class="col-sm-4">
-        <h4>Mountain Excursion</h4>
-        <p>Expedition excursion design darn excursion fun, clean simple organized WordPress Travel colorful webdesign. Traveler blogger website design expedition clean excursion traveling.</p>
-        <button type="button" class="btn btn-info">Get More Info</button>
+      <div class="row">
+          <div class="col-sm-6">
+              <img src="img/fun.jpg" class="img-fluid rounded" alt="Lets have fun!">
+          </div>
+          <div class="col-sm-6">
+            <h2>Adventure Awaits</h2>
+            <p>Adventure webdesign pretty design design, excursion cute WordPress blogger design webdesign adventure. Expedition colorful design simple excursion blogger blogger design WordPress design, design organized website theme.</p> 
+            <p>Pretty simple traveling fun WordPress wanderlust darn simple organized. WordPress travelblogger simple WordPress cute whimsical organized. Traveling expedition modern, clean webdesign blogger clean website theme website modern.</p>
+          </div>
       </div>
-      <div class="col-sm-4">
-        <h4>Island Getaway</h4>
-        <p>Expedition excursion design darn excursion fun, clean simple organized WordPress Travel colorful webdesign. Traveler blogger website design expedition clean excursion traveling.</p>
-        <button type="button" class="btn btn-info">Get More Info</button>
+      <!-- JavaScript Component: Carousel -->
```

Comment:
Nice carousel and comment. Something small like this comment can make a difference. It showcases to employers your communication and organization skills without having to say a word!